### ISSUE TITLE
Check if GPODDER_DOWNLOAD_DIR is mounted on launch (and is writeable)

### ISSUE THREAD

**Issue by rmatelec** [gpodder/gpodder#337]
Hi, I have the following setup:

1. gpodder refreshes subscription feeds on startup
2. GPODDER_DOWNLOAD_DIR is set to a external disk that may or may be not mounted on startup (my media library get bigger every day :)) 

Sometimes, I forgot to plug the external drive before launching gpodder, resulting in the deletion of the missing older episodes from the sqlite database because they are old and the file is missing. I mitigated this issue launching gpodder with a script that backups the Database file before start (so i can plug the disk, restore the database, keep all the information in sync and get the new episodes), but i thing would be nice if gpodder check this for me, as I think is not such a rare use case.

Thanks for this great piece of software!

**Comment by adamvoss** [comment]
I agree this does not seem like very user friendly behavior.  As a workaround, you could set your entire GPODDER_HOME to be on the external drive, then gPodder will fail launching earlier and not have access to the database to delete episodes (this is how I am set up).
